<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DFRobot - Remote Controller</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="connectivityGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Connectivity</string>
    </property>
    <widget class="QLineEdit" name="ipTextBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLabel" name="ipLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="portLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="portTextBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>50500</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="motorsControlGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>191</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Motors Control</string>
    </property>
    <widget class="QSlider" name="m1slider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>21</width>
       <height>181</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="m1label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>M1</string>
     </property>
    </widget>
    <widget class="QLabel" name="m4label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>M4</string>
     </property>
    </widget>
    <widget class="QLabel" name="m3label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>M3</string>
     </property>
    </widget>
    <widget class="QLabel" name="m2label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>M2</string>
     </property>
    </widget>
    <widget class="QLabel" name="m1speedLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="m2slider">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>21</width>
       <height>181</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="m3slider">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>21</width>
       <height>181</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="m4slider">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>21</width>
       <height>181</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="m2speedLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="m3speedLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>230</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="m4speedLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="m1DirButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="m2DirButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="m3DirButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="m4DirButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="sendHelloButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>85</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Send hello!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>m1DirButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateServer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m2DirButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateServer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m3DirButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateServer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m4DirButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateServer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateServer()</slot>
 </slots>
</ui>
